src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -85,7 +89,7 @@ function isolateAppeal(state, id) {
 function mapStateToProps(state, ownProps) {
   return {
     appeal: isolateAppeal(state, ownProps.params.id),
-    loading: state.loading
+    appealsLoading: state.disability.status.appeals.appealsLoading
```

Comment:
So I take it loading status was already being stored in redux?

Reply:
Yessir!